core/code/portal_detail_display_tools.js

```diff
@@ -2,6 +2,23 @@
 // hand any of these functions the details-hash of a portal, and they
 // will return pretty, displayable HTML or parts thereof.
 
+window.getPortalHistoryDetails = function (d) {
+  if (!d.history) {
+    return ('<div style="text-align: center; color: #ffce00">History missing</div>')
+  }
+
+  var visited = d.history.visited;
+  var captured = d.history.captured;
+  var scouted = d.history.scanned;
```

Comment:
```suggestion
  var scoutControlled = d.history.scoutControlled;
```

Reply:
Attention!
At the moment 'scanned' layer is wrongly titled as 'Scout Controlled' in stock intel!
"Scout Controller" is a volatile status, that the agent looses sooner or later on interaction of other agents with the same portal.
"scanned" is a permanent status, set when the agent submits the first 3D-Scan of a portal.
@MysticJay

Reply:
from field data, the data `scanned` or `scoutControlled` is set to 1 if the player gains the scout controlled at one point.
The fact that one gains scout controlled is permanent and counted in the agent stats for the "Scout Controller" medal.

This follows the same principle as `captured`, one looses the own of a portal but the capture stays